.github/workflows/pr_push.yml

```diff
@@ -113,3 +115,6 @@ jobs:
   ProxyLib:
     needs: [Build]
     uses: ./.github/workflows/proxy_lib.yml
+  GPU:
+    needs: [Build]
+    uses: ./.github/workflows/gpu.yml
```

Comment:
We may add badges for new workflows later ...

Reply:
added